Tools/baselineDef.cc

```diff
@@ -2041,6 +2044,21 @@ void BaselineVessel::PassHEMVeto()
     tr->registerDerivedVar("SAT_Pass_HEMVeto"+firstSpec,   SAT_Pass_HEMVeto);
 }
 
+void BaselineVessel::GetPileup()
```

Comment:
what do I do?  I return nothing?

Reply:
Originally I was planning to register puWeight2017weightedAverage. However, I first printed out puWeight2017weightedAverage as well as puWeight and compared their values, which are close. Then I decided that we can just use puWeight.

Reply:
This function is just for testing.